OBSDOCS-961: Update docs to explain persistence of silences when mult… (#75013)
openshift/openshift-docs: modules/monitoring-configuring-persistent-storage.adoc

// * observability/monitoring/configuring-the-monitoring-stack.adoc

:_mod-docs-content-type: CONCEPT
[id="configuring-persistent-storage_{context}"]
= Configuring persistent storage

Run cluster monitoring with persistent storage to gain the following benefits:

> * Protect your metrics and alerting data from data loss by storing them to a persistent volume (PV). As a result, they can survive pods being restarted or recreated.

Reviewer: ```suggestion
* Protect your metrics and alerting data from data loss by storing them to a persistent volume (PV). 
```

Reviewer: Not really sure if the last sentence is needed? If you feel it is, may need to be re-worded.

Reviewer 2: I'd leave that to https://issues.redhat.com/browse/OBSDOCS-547 (where we're reshaping that page),but sure we can remove that as users should be more familiar with PV, and we have a dedicated page for that now.

> * Protect your metrics and alerting data from data loss by storing them to a persistent volume (PV). As a result, they can survive pods being restarted or recreated.

Author: I think both would be okay, but I think you are right as "in" is more natural in this case.

> * Protect your metrics and alerting data from data loss by storing them to a persistent volume (PV). As a result, they can survive pods being restarted or recreated.

Author: @machine424 WDYT?

* Avoid getting duplicate notifications and losing silences for alerts when the Alertmanager pods are restarted.

> For production environments, it is highly recommended to configure persistent storage. Because of the high IO demands, it is advantageous to use local storage.

Reviewer: ```suggestion
For production environments, it is highly recommended to configure persistent local storage.

Reviewer: Not sure if the sentences above could be combined as they are both recommended?

Author: I did not want to touch the last sentence, because it is going to be deleted as part of another issue :D

Author: (We do not want to recommend using local storage anymore)

// * observability/monitoring/configuring-the-monitoring-stack.adoc

:_mod-docs-content-type: CONCEPT
[id="configuring-persistent-storage_{context}"]
= Configuring persistent storage

Run cluster monitoring with persistent storage to gain the following benefits:

* Protect your metrics and alerting data from data loss by storing them to a persistent volume (PV). As a result, they can survive pods being restarted or recreated.

> * Avoid losing silences for alerts when the Alertmanager pods are restarted.
> * Avoid duplicate notifications for cluster and node upgrades.

Reviewer 2: ```suggestion
* Avoid getting duplicate notifications and losing silences for alerts when the Alertmanager pods are restarted.
```
both of them are about AM Pods.
